setup.py

```diff
@@ -1,18 +1,13 @@
 #!/usr/bin/env python
 # -*- encoding: utf-8 -*-
-from __future__ import absolute_import
-from __future__ import print_function
+from __future__ import absolute_import, print_function
```

Comment:
If we don't care about support Python 2, these could go.

Reply:
Indeed! @pgleeson given Python 2 is retiring next month, do we still want to put work into supporting it?

Reply:
Interestingly, it appears from #13 that the latest pynwb has dropped support for Python 2...